### PR TITLE
CIV-8843 evidence upload fast track upload documents validation error 3 file upload

### PR DESCRIPTION
**Pull request by LeonardoPalmeiro**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-8843

### Change description ###

image

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/routes/features/caseProgression/typeOfDocumentsController.ts [hunk 1]
```diff
@@ -27,8 +27,8 @@ async function renderView(res: Response, claimId: string, form: GenericForm<Uplo
   const claim = await getCaseDataFromStore(claimId);
   const claimantFullName = claim.getClaimantFullName();
   const defendantFullName = claim.getDefendantFullName();
-  const isFastTrack = claim.isFastTrackClaim;
-  const isSmallClaims = claim.isSmallClaimsTrackDQ;
+  const isFastTrack = true;
```
Comment (hmcts-version1-ignacio) [inline src/main/routes/features/caseProgression/typeOfDocumentsController.ts:30]: Did you forget to remove this?

**Review by hmcts-version1-ignacio — APPROVED** [review]

File: src/main/routes/features/caseProgression/uploadFileController.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+const multer = require('multer');
+const storage = multer.memoryStorage();
+const upload = multer({storage: storage});
+
+const uploadFileController = Router();
+
+uploadFileController.post(CP_UPLOAD_FILE, upload.single('file'), (req, res) => {
+  try {
+    //const claimId = req.params.id;
```
Comment (kurt-uk) [inline src/main/routes/features/caseProgression/uploadFileController.ts:16]: dont need these comments in

**Review by kurt-uk — APPROVED** [review]

File: src/test/utils/errorMessageTestConstants.ts [hunk 3]
```diff
@@ -113,6 +113,12 @@ export const enum TestMessages {
   VALID_DEFENDANT_AGE_ELIGIBILITY = 'Select yes, no, or company/organisation',
   VALID_ENTER_A_DATE_BEFORE = 'Please enter a date of birth before',
   VALID_ENTER_TYPE_OF_DOCUMENT = 'You must enter type of document',
+  VALID_ENTER_WITNESS_NAME = 'You must enter name of witness',
+  VALID_ENTER_EXPERT_NAME = 'You must enter name of expert',
+  VALID_ENTER_EXPERT_NAMES = 'You must enter the name of at least one expert',
+  VALID_ENTER_EXPERTISE = 'You must enter field of expertise',
+  VALID_ENTER_DOCUMENT_QUESTIONS = 'You must enter name of document you have questions about',
+  VALID_ENTER_DOCUMENT_QUESTIONS_OTHER_PARTY = 'You must enter name of document with other party&#39;s questions',
```
Comment (hmcts-version1-ignacio) [inline src/test/utils/errorMessageTestConstants.ts:121]: Just some weird symbols to remove: "party&#39;s"

File: src/main/common/form/validators/isAllowedMimeType.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  ValidationArguments, ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface,
+} from 'class-validator';
+
+@ValidatorConstraint({ name: 'isAllowedMimeType', async: false })
+export class IsAllowedMimeTypeValidator implements ValidatorConstraintInterface {
+  validate(value: any, args: ValidationArguments) {
+    const allowedMimeTypes: string[] = [
+      'application/vnd.openxmlformats-officedocument.wordprocessingml.document', // Word
```
Comment (LeonardoPalmeiro) [inline src/main/common/form/validators/isAllowedMimeType.ts:12]: we need to add 
application/msword
application/vnd.ms-excel
application/vnd.ms-powerpoint

**Pull request merged**